index.html

```diff
@@ -41,5 +41,141 @@ <h4>LET'S CONNECT</h4>
+      <img src="./images/Text-2.svg" alt="text">
+    </div>
+    <div class="main-skill">
+      <ul>
+      <li><img src="./images/html.svg" alt="html"></li>
+      <li><img src="./images/css.svg" alt="css"></li>
+      <li><img src="./images/jS.svg" alt="javaScript"></li> 
+    </ul> 
+    </div>
+    <div class="button-1">
+      <form action="./index.html">
+        <button type="submit">See Project</button>
+      </form>
+    </div>
+  </section>
+  <section class="project-1">
+    <div class="main-image">
+      <img src="./images/Portfolio-2.svg" alt="mainimage">
+    </div>
+    <div class="main-headline">
+      <img src="./images/project-title-2.svg" alt="projectname">
+    </div>
+    <div class="main-role">
+      <img src="./images/frame2.svg" alt="role">
+    </div>
+    <div class="main-text">
+      <img src="./images/Text-2.svg" alt="text">
+    </div>
+    <div class="main-skill">
+      <ul>
+      <li><img src="./images/html.svg" alt="html"></li>
+      <li><img src="./images/css.svg" alt="css"></li>
+      <li><img src="./images/jS.svg" alt="javaScript"></li> 
+    </ul> 
+    </div>
+    <div class="button-1">
+      <form action="./index.html">
+        <button type="submit">See Project</button>
+      </form>
+    </div>
+  </section>
+
+  <section class="project-1">
+    <div class="main-image">
+      <img src="./images/portfolio-3.svg" alt="mainimage">
+    </div>
+    <div class="main-headline">
+      <img src="./images/ProjectTitle.svg" alt="Tonic">
+    </div>
+    <div class="main-role">
+      <img src="./images/frame2.svg" alt="role">
+    </div>
+    <div class="main-text">
+      <img src="./images/Text-2.svg" alt="text">
+    </div>
+    <div class="main-skill">
+      <ul>
+      <li><img src="./images/html.svg" alt="html"></li>
+      <li><img src="./images/css.svg" alt="css"></li>
+      <li><img src="./images/jS.svg" alt="javaScript"></li> 
+    </ul> 
+    </div>
+    <div class="button-1">
+      <form action="./index.html">
+        <button type="submit">See Project</button>
+      </form>
+    </div>
+  </section>
+  <section class="project-1">
+    <div class="main-image">
+      <img src="./images/portfolio-4.svg" alt="mainimage">
+    </div>
+    <div class="main-headline">
+      <img src="./images/project-title-2.svg" alt="Tonic">
+    </div>
+    <div class="main-role">
+      <img src="./images/frame2.svg" alt="role">
+    </div>
+    <div class="main-text">
+      <img src="./images/Text-2.svg" alt="text">
+    </div>
+    <div class="main-skill">
+      <ul>
+      <li><img src="./images/html.svg" alt="html"></li>
+      <li><img src="./images/css.svg" alt="css"></li>
+      <li><img src="./images/jS.svg" alt="javaScript"></li> 
+    </ul> 
+    </div>
+    <div class="button-1">
+      <form action="./index.html">
+        <button type="submit">See Project</button>
+      </form>
+    </div>
+  </section>
+  <section class="showcase-2">
+    <h1>About <br>Myself</h1>
```

Comment:
- [ ] It's important to follow semantics and best practices to improve our projects as much as we can! The `h1` is used as the main title of a website because represents the main heading/subject for the whole page so just used it once through out your website😉 You could use from `h2` to `h6` tags in a hierarchical manner with the rest of your texts 📝